testsuite/ui/widgets/buttons.py

```diff
@@ -23,25 +23,33 @@ class ThreescaleButton(widgetastic_patternfly4.Button):
     therefore a class with custom locator is needed.
     """
 
-    def __init__(self, parent, text, classes, **kwargs):
+    def __init__(self, parent, classes, text=None, element_id=None, **kwargs):
```

Comment:
Order of commits is reversed, the commit with update of button should be first. Actually even separate PR would be nice however it is not necessary. Certainly all the button relevant changes should be in one commit (not the case now).

According to logic of widgetastic_patternfly4.Button `element_id` should be just `id` actually (pylint may complain, I don't know, anyway if so, it's good to mute it). Actually, again according to behavior of Button superclass, `id` does not have to be listed in args, all the kwargs should be part of locator or gather `elif 'id' in kwargs:` instead of `elif element_id`.

This signature will not break the interface (less lines to change):
```
def __init__(self, parent, text=None, *, classes, **kwargs):
```

and still should be compatible with both new and old usage (unless `classes` arg is used somewhere without keyword; I didn't find anything like this, anyway even in such case changing that usage would be more preferable than current proposal as it would preserve interface of supeclass Button)